### ISSUE TITLE
Change CLI default logging level

### ISSUE THREAD

**Issue by jelovirt** [dita-ot/dita-ot#3740]
## Description
Change `dita` command (CLI) default logging level from _warning_ to _info_ and add option `-q, --quiet` to enable _warning_ level logging.

## Additional Context
The traditional UNIX command line convention has been to not print anything to standard out if there are not errors. DITA-OT CLI logging levels have been:

| logging option | Ant logging level |
|---|---|
| _default_ | warning |
| `-v` | info |
| `-d` | verbose |

Given that most of our users are likely not tranditional UNIX command line users and we should prioritize clarity, a more user friendly logging levels would be:

| logging option | Ant logging level |
|---|---|
| `-q` | warning |
| _default_ | info |
| `-v`, `-d` | verbose |

Ant has a very low level logging level _debug_, but we should not enable that in CLI output because it's intended and used only by Ant for debugging and behaviour. Thus, `-v` and `-d` should become synonyms.

**Comment by hestonhoffman** [comment]
This would be a welcome change. I currently have to run a separate logging script after each build that pulls out INFO lines. 

It's especially problematic for markdown transformations where we get a lot of the following (because of unescaped `[]` and `<>` characters in our code base, where the markdown is generated from):
```
[DOTJ047I][INFO] Unable to find key definition for key reference "cert regen docs" in root scope. The href attribute may be used as fallback if it exists
```

**Comment by jelovirt** [comment]
Instead of changing the info level default, I wonder why DOTJ047I is not logged as warning.

**Comment by hestonhoffman** [comment]
Yeah, that would be ideal, but perhaps is cause for a new ticket?

**Comment by xephon2** [comment]
@jelovirt I have reported this in #2793 in 2017.
